.github/reviewers.yaml

```diff
@@ -0,0 +1,37 @@
+reviewers:
+  # The default reviewers
+  defaults:
+    - maintainers
+
+  groups:
+    maintainers:
+      - nrb
+      - ashish-amarnath
+      - carlisia
+      - zubron
+      - dsu-igeek
+      - jenting
+
+    tech-writer:
+      - a-mccarthy 
+
+files:
+  'public/**':
+    - tech-writer
+  '**/*.md':
+    - tech-writer
+  '/design/**': # Technical design requests are ".md" files but should be reviewed by maintainers
+    - maintainers
```

Comment:
What about this one?

Reply:
This one should pick from the list of maintainers exclusively.